internal/tfimport/tfimport.go

```diff
@@ -656,28 +723,29 @@ func planAndImport(rn, importRn runner.Runner, runArgs *RunArgs, skipped map[str
 		}
 
 		// Handle the different outcomes of the import attempt.
-		var ie *importer.InsufficientInfoErr
+		var iie *importer.InsufficientInfoErr
 		var se *importer.SkipErr
 		switch {
 		// err will only be nil when the import succeed.
 		// Import succeeded, print the success output.
 		case err == nil:
 			// Import succeeded.
-			importedSomething = true
+			log.Printf("Successfully imported %v\n", cc.Address)
+			successes = append(successes, cc.Address)
 
 		// Check if the user manually skipped the import.
 		case errors.As(err, &se):
 			skipped[cc.Address] = true
 
 		// Check if the error indicates insufficient information.
-		case errors.As(err, &ie):
-			log.Printf("Insufficient information to import %q\n", cc.Address)
+		case errors.As(err, &iie):
+			log.Printf("Insufficient information to import %q, missing fields %s\n", cc.Address, strings.Join(iie.MissingFields, ","))
 
-			errMsg := fmt.Sprintf("%q (insufficient information)", cc.Address)
+			failedMsg := fmt.Sprintf("%v (insufficient information)", cc.Address)
```

Comment:
i would revert these var renames. failedMsg just seems like a longer way of saying errMsg